spec/spec_helper.rb

```diff
@@ -5,6 +5,22 @@
 require "factory_girl_rails"
 require "support/factory_girl"
 
+OmniAuth.config.test_mode = true
+
+OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new({
+  :provider => 'github',
+  :uid => '123545',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.